### ISSUE TITLE
Proxy LocalAPI through Modbus

### ISSUE THREAD

**Issue by jjcasmar** [gpulido/python-airzone#21]
Would it be possible to expose a webserver that exposes the same API as the LocalAPI which communicates with Modbus? That way, Airzone devices that don't support LocalAPI could be used with LocalAPI integrations (home assistant for example) via the proxy.

**Comment by gpulido** [comment]
Hello,
It is a very interesting idea.
Is there an official localapi reference to be used?
(In the mean time for  modbus in HA. It is already supported by https://github.com/gpulido/homeassistant-airzone)

**Comment by jjcasmar** [comment]
Yeah, I am actually using that integration in HomeAssistant, but since Airzone provides/support (or at least is linked from Airzone website) the official integration, would be interesting to be able to use directly that one, exposing a fake LocalAPI. 

There is a reference manual https://doc.airzone.es/producto/Gama_AZ6/Airzone/Comunes/Manuales/MI_AZ6_WSCLAPI_A4_MUL.pdf

**Comment by jjcasmar** [comment]
If you need any help testing this, I currently have a RPi connected to the Modbus port of my Aidoo device, so I can definitely test it.

**Comment by Noltari** [comment]
@jjcasmar @gpulido I've just seen this issue and I think that the Local API simulator that I've implemented in the official integration library repository could be helpful for this purpose:
https://github.com/Noltari/aioairzone/blob/master/sim/airzone.py

You can also use the following resource for the Local API documentation:
https://developers.airzonecloud.com/docs/local-api/
